autotst/calculator/statmech.py

```diff
@@ -554,6 +686,211 @@ def run(self):
+        This rule of thumb was taken from the textbook: Section 2.9 of `Thermochemical 
+        Kinetics: Methods for the Estimation of Thermochemical Data and Rate Parameters` 
+        by Sidney W. Benson.
+        
+        
+        Inputs:
+        - conformer (autotst.species.Conformer): the conformer of interest
+        - torsion_index (int): the index corresponding to the rotor of interest
+        
+        Returns:
+        - estimated_barrier (float): the estimated barrier to rotation in kcal
+        """
+        
+        # Keys are number of substituents
+        # Values are barriers in kcal
+        ESTIMATED_BARRIERS = {
+            0: 0,
+            1: 1.1,
+            2: 2.2,
+            3: 3.5
+        }
+        
+        torsion = conformer.torsions[torsion_index]
+        i,j,k,l = torsion.atom_indices
+        j_neighbors = len(conformer.rmg_molecule.atoms[j].bonds) - 1
+        k_neighbors = len(conformer.rmg_molecule.atoms[k].bonds) - 1
+    
+        return ESTIMATED_BARRIERS[min([j_neighbors, k_neighbors])]
+
+    def find_closest_vibrational_arrhenius(self, w):
+        """
+        A method that will find the arrhenius expression corresponding to 
+        the vibrational contribution to the rate expression. This will 
+        build the linear interplation if need be and use that interpolation
+        to find values of A, Ea, and n. 
+
+        Inputs:
+        - w (float): vibration in cm^-1
+
+        Returns:
+        - arrhenius (rmgpy.kinetics.Arrhenius): the vibration contribution 
+            to the rate expression
+        """
+        if not self._vib_interpolation:
+            self._vib_interpolation = build_vibrational_interpolation()
+        logA, n, B = self._vib_interpolation(w)
+        A = 10**logA * 1000
+        Ea = B / constants.R
+        return Arrhenius(A=(A, 'cm^3/(mol*s)'), n=n, Ea=(Ea, 'J/mol'))
+
+    def find_closest_hindered_arrhenius(self, V, Q):
+        """
+        A method that will find the Arrhenius expression corresponding to 
+        the hindered rotor contribution to the rate expression. This will
+        build the linear interpolations if need be and use those to find 
+        values of A, Ea and n.
+
+        Inputs:
+        - V (float): the barrier to rotation in kcal/mol
+        - Q (float): the free rotor partiton function
+
+        Returns:
+        - arrhenius (rmgpy.kinetics.Arrhenius): the hindered rotor 
+            contribution to the rate expression
+        """
+        if not self._hr_interpolation:
+            self._hr_interpolation = build_hr_interpolations()
+
+        logQ = np.log10(Q)
+        logA = self._hr_interpolation[0](V,logQ)[0]
+        n = self._hr_interpolation[1](V,logQ)[0]
+        B = self._hr_interpolation[2](V,logQ)[0]
+        A = 10**logA * 1000
+        Ea = B / constants.R
+        return Arrhenius(A=(A, 'cm^3/(mol*s)'), n=n, Ea=(Ea, 'J/mol'))
```

Comment:
I guess it might cancel out if you divide by a vibrational_arrhenius every time you multiply by a hindered_rotor_arrhenius, but shouldn't these A factors be dimensionless?  (because they're always multiplied or divided by the actual k)